zhaquirks/sonoff/trvzb.py

```diff
@@ -121,5 +136,33 @@ def _is_manuf_specific(self):
         fallback_name="Valve closing degree",
         initially_disabled=True,
     )
+    .number(
+        CustomSonoffCluster.AttributeDefs.temperature_control_accuracy.name,
+        CustomSonoffCluster.cluster_id,
+        min_value=-1.0,
+        max_value=-0.2,
+        step=0.2,
+        unit=UnitOfTemperature.CELSIUS,
+        multiplier=0.01,
+        translation_key="temperature_control_accuracy",
+        fallback_name="Temperature control accuracy",
+    )
+    .switch(
+        CustomSonoffCluster.AttributeDefs.external_temperature_sensor_enable.name,
+        CustomSonoffCluster.cluster_id,
+        translation_key="external_temperature_sensor_enable",
```

Comment:
I'll remove the `_enable` suffix from this `translation_key`. More in line with other translation keys in ZHA/quirks and HA then.
```suggestion
        translation_key="external_temperature_sensor",
```